scipy/special/tests/test_data.py

```diff
@@ -489,11 +490,8 @@ def test_local():
         data_local(ellip_harm, 'ellip',(0, 1, 2, 3, 4), 5, rtol=1e-10, atol=1e-13),
     ]
 
-    with warnings.catch_warnings():
-        warnings.simplefilter("ignore", category=IntegrationWarning)
-
-        for test in TESTS:
-            yield _test_factory, test
+    for test in TESTS:
+        yield _test_factory, test
```

Comment:
Running pytest on just this file, after creating an empty `__init__.py` in the test directory, did not give me any issues with `ellip_harm_2` or _ellip_harm`. Can you give a simple way to reproduce this? The `sph*` functions are gone with #7601.